dota2/features/lobby.py

```diff
@@ -0,0 +1,221 @@
+
+        command = {
+            "lobby_details": options,
+            "pass_key": password
+        }
+        if tournament_game_id > 0:
+            command.update({
+                "tournament_game": True,
+                "tournament_game_id": tournament_game_id,
+                "tournament_id": tournament_id
+            })
+        jobid = self.send_job(EDOTAGCMsg.EMsgGCPracticeLobbyCreate, command)
+
+        @self.once(jobid)
+        def wrap_create_tournament_lobby(message):
+            self.emit('create_tournament_lobby', message)
+
+        return jobid
+
+    def config_practice_lobby(self, lobby_id, options=None):
+        options = {} if options is None else options
+        raise NotImplementedError()
+
+    def request_practice_lobby_list(self):
+        raise NotImplementedError()
+
+    def request_friend_practice_lobby_list(self):
+        raise NotImplementedError()
+
+    def balanced_shuffle_lobby(self):
+        raise NotImplementedError()
+
+    def flip_lobby_teams(self):
+        raise NotImplementedError()
+
+    def invite_to_lobby(self, steam_id):
+        """
+        Asks to invite a player to your lobby. This creates a new default lobby when you are not already in one.
+
+        :param steam_id: steam_id
+        :type steam_id: :class:`int`
+        :return: job event id
+        :rtype: str
+
+        Response event: ``create_invite_to_lobby``
+
+        :param message: `CMsgInviteToLobby <https://github.com/ValvePython/dota2/blob/ca75440adca20d852b9aec3917e4387466848d5b/protobufs/base_gcmessages.proto#L88>`_ proto message
+        """
+        import ipdb; ipdb.set_trace()
+        if self.verbose_debug:
+            self._LOG.debug("Inviting %s to a lobby." % steam_id)
+
+        jobid = self.send_job(EGCBaseMsg.EMsgGCInviteToLobby, {
+            "steam_id": steam_id
+        })
+
+        @self.once(jobid)
+        def wrap_create_invite_to_lobby(message):
+            self.emit('create_invite_to_lobby', message)
+
+        return jobid
+
+    def practice_lobby_kick(self, account_id):
+        raise NotImplementedError()
+
+    def practice_lobby_kick_from_team(self, account_id):
+        raise NotImplementedError()
+
+    def join_practice_lobby(self, id, password):
+        raise NotImplementedError()
+
+    def leave_practice_lobby(self):
+        """
+        Sends a message to the Game Coordinator requesting to leave the current lobby.
+
+        :return: job event id
+        :rtype: str
+
+        Response event: ``leave_practice_lobby``
+
+        :param message: `CMsgPracticeLobbyLeave <https://github.com/ValvePython/dota2/blob/ca75440adca20d852b9aec3917e4387466848d5b/protobufs/dota_gcmessages_client_match_management.proto#L181>`_ proto message
+        """
+        if self.verbose_debug:
+            self._LOG.debug("Sending match CMsgPracticeLobbyLeave request")
+
+        jobid = self.send_job(EDOTAGCMsg.EMsgGCPracticeLobbyLeave, {})
```

Comment:
Is this really a job? Do you actually get a response back? I'm guessing not. So you can lose the all the wrap stuff. That goes for other places where it's the same.